can-map.js

```diff
@@ -318,12 +318,12 @@ var Map = Construct.extend(
 				var first = attr.substr(0, dotIndex),
 					second = attr.substr(dotIndex+1);
 
-				current =  this.__inSetup ? undefined : this.___get( first );
+				current =  this[canSymbol.for("can.initializing")] ? undefined : this.___get( first );
```

Comment:
You should cache the symbol once at the top of the module.

Reply:
👍 Sure thing